merlin/analysis/generatemosaic.py

```diff
@@ -165,6 +167,11 @@ def _prepare_mosaic_slice(self, zIndex, dataChannel, micronExtents,
                 inputImage[:, :cropWidth] = 0
                 inputImage[:, inputImage.shape[0] - cropWidth:] = 0
 
+            if self.parameters['draw_fov_labels']:
```

Comment:
It'd be preferable to place the labels based on the relative position as opposed to absolute, in case someone uses a smaller format camera.